test/functional/legacy/003_cindent_spec.lua

```diff
@@ -653,8 +654,8 @@ describe('cindent', function()
       }
 
       Constructor::Constructor(int a,
-                               int b 
-                              )  : 
```

Comment:
this text is a test fixture, it should not be linted

Reply:
I'll remove this change, but could you kindly explain what a "test fixture" is? My engrish is lacking...

Reply:
"test fixture" is any resource needed to set up a test, e.g. sample text, a fake file, a fake plugin, etc.